mysite/_config/config.yml

```diff
@@ -1,21 +1,11 @@
 ---
-Name: mysiteconfig
-After:
-  - 'framework/*'
-  - 'cms/*'
+Name: mycwpsite
 ---
-# YAML configuration for SilverStripe
-# See http://doc.silverstripe.org/framework/en/topics/configuration
-# Caution: Indentation through two spaces, not tabs
-SiteTree:
-  extensions:
-    - WorkflowEmbargoExpiryExtension
-SSViewer:
-  theme: 'starter'
-MySQLDatabase:
-  connection_charset: 'utf8'
-LeftAndMain:
+SilverStripe\View\SSViewer:
+  themes:
+    - 'starter'
+    - '$default'
```

Comment:
We should remove this and add a `theme.yml` like silverstripe/installer: https://github.com/silverstripe/silverstripe-installer/blob/4.0/mysite/_config/theme.yml